### ISSUE TITLE
The label for and id for the pair must be identical in Wizards tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1918]
URL: http://clarity-ks.surge.sh/wizards

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Wizards tab.
3. Click OPEN WIZARD button for Ghost pages.
4. Use WAT to check label for and ID property.

Actual Results:
The “label for” associating "Page categories” text within Step 1 is label for="typesOfPages".
The “id” associating "Select pages you want to show” within Step 1 is id="typesOfPages11".
Overall, the “label for” and “id” for the pair are different.
For more details, please refer to " Wizards-LabelForAndIdAreDifferent.zip".

Expected Results:
The label for and id for the pair must be identical.

Fix Suggestion:
Set label for property to “label for="typesOfPages11"” for "Page categories” text.

Violation of Section 508 Standards: 
1194.22 (n) When electronic forms are designed to be completed on-line, the form shall allow people using assistive technology to access the information, field elements, and functionality required for completion and submission of the form, including all directions and cues.

User Impact: Blind.

Screen shot:
1.	Wizards-LabelForAndIdAreDifferent.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/66855231-23f4-4930-829e-cada25a7f41d)

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/96fd1be6-4802-4e46-aba8-90967d9fc7c0)

NOTE: This may be more involved as it might require projecting those items into the wizard steps

**Comment by Jinnie** [comment]
Ghost pages have been removed in #2152
The remaining forms on the wizards page look ok.
